chukwuemekachm/M-Tracker-React-Client
# #161121505 Completes admin approve, disapprove and resolve request feature

#### What does this PR do?
- This PR completes admin approve, disapprove and resolve request feature
#### Description of Task to be completed?
- Add modifyRequestStatus to adminRequestActions
- Modify dashboard.jsx
- Modify ViewSingleRequestPage.jsx
- Modify tests
- Update snapshots
#### How should this be manually tested?
- Clone the repo
- Start up command line
- Navigate to the project directory using `cd`
- Run `npm install`
- Run `npm start` or `npm run dev`
- Your default browser should be opened if you used `npm run dev`
- Login or signup(navigate to `/login` or `/signup`)
- You will be redirected to the dashboard
- Click on requests with different status and preview them
#### Production URL
- [heroku](https://m-tracker-react.herokuapp.com)
#### Any background context you want to provide?
- None
#### What are the relevant pivotal tracker stories?
- [#161121505](https://www.pivotaltracker.com/story/show/161065878)
#### Screenshots (if appropriate)

<img width="1440" alt="screen shot 2018-10-12 at 11 10 50 am" src="https://user-images.githubusercontent.com/33798252/46864244-b0c0ee80-ce11-11e8-83ad-259f7e1c18ec.png">


<img width="1440" alt="screen shot 2018-10-12 at 11 10 59 am" src="https://user-images.githubusercontent.com/33798252/46864245-b1598500-ce11-11e8-8be3-557232291077.png">

#### Questions:
- None
